README.md

```diff
@@ -103,6 +105,13 @@ Pay.setup do |config|
 end
 ```
 
+## Emails
+
+Emails can be enabled/disabled using the `send_emails` configuration option (enabled per default). When enabled, the following emails will be sent:
+
+- When a charge succeeded
+- When a charge was refunded
```

Comment:
Just realized, since Braintree doesn't have webhooks for charges, we'll need to probably send emails directly for those. 

https://developers.braintreepayments.com/guides/webhooks/overview